.github/workflows/ci.yml

```
@@ -0,0 +1,65 @@
+      MONGODB_URI: ${{ secrets.MONGODB_URI }}
+      API_KEY: ${{ secrets.API_KEY }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Debug environment variables
+        run: |
+            echo "MONGODB_URI: $MONGODB_URI"
+            echo "API_KEY: $API_KEY"
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)